src/Tribe/Views/V2/Assets.php

```diff
@@ -45,5 +45,14 @@ public function register() {
 			[ 'jquery', 'tribe-common', 'tribe-query-string', 'underscore' ],
 			null // prevent it from loading
 		);
+
+		tribe_asset(
+			$plugin,
+			'tribe-events-views-v2-tooltip',
+			'views/tooltip.js',
+			[ 'jquery', 'tribe-common', 'tribe-events-views-v2-manager', 'tribe-tooltipster' ],
+			'wp_enqueue_scripts',
```

Comment:
We should have logic here to only load this in the correct pages. I mean on all of the new V2 views.